extensions/quarkiverse-playwright.yaml

```diff
@@ -0,0 +1,6 @@
+---
+group-id: "io.quarkiverse.playwright"
+artifact-id: "quarkus-playwright"
+versions:
+- "2.0.0"
+exclude-versions: []
```

Comment:
Add the previous versions here, because they don't have a YAML file

Reply:
Done!

Reply:
Sorry, I meant exclude the versions, 2.0.0 is the only valid version AFAIK